oie/ja-jp/content/topics/provisioning/azure/haad-join/prereqs-haad.htm

```diff
@@ -381,10 +381,10 @@ <h2><a name="Prerequi2"></a>Okta環境の前提条件</h2>
                                                                     <p>シングルサインオンの構成時に、<span class="uicontrol">WS-Federation（自動または手動）</span>の方法を選択します。</p>
                                                                 </li>
                                                                 <li>
-                                                                    <p>プロビジョニングにOktaを使用している場合は、<span class="uicontrol">プロビジョニングタイプ</span>を<span class="uicontrol">Licenses/Role Management Only</span>または<span class="uicontrol">Profile Sync</span>に設定します。ほかのオプション（［User Sync（ユーザーの同期）］と［Universal Sync（ユニバーサル同期）］）は、ハイブリッドのセットアップに必要なAzure AD Connectと互換性がありません。</p>
+                                                                    <p>プロビジョニングにOktaを使用している場合は、<span class="uicontrol">プロビジョニングタイプ</span>を<span class="uicontrol">Licenses/Role Management Only</span>または<span class="uicontrol">Profile Sync</span>に設定します。ほかのオプション（［ユーザー同期］と［ユニバーサル同期］）は、ハイブリッドのセットアップに必要なAzure AD Connectと互換性がありません。</p>
                                                                 </li>
                                                                 <li>
-                                                                    <p>プロビジョニング中にプロファイルマッピングを構成する際に、ユーザーがデバイスへのサインインに使用するユーザー名と一致するように<span class="uicontrol">アプリケーションユーザー名のフォーマット</span>フィールドを設定します。通常、これはAzure ADユーザープリンシパル名です。</p>
+                                                                    <p>プロビジョニング中にプロファイルマッピングを構成する際に、ユーザーがデバイスへのサインインに使用するユーザー名と一致するように<span class="uicontrol">［アプリケーションユーザー名のフォーマット］</span>フィールドを設定します。通常、これはAzure ADユーザープリンシパル名です。</p>
```

Comment:
Not a regression, this is missing the EN portion already. We have many of these, it now seems.